styles/old/layout/main.css

```diff
@@ -1,19 +1,33 @@
 #main {
   .container {
-    padding: 0 20px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  .has-side-nav.container {
+    display: grid;
+    grid-template-columns: fit-content(300px) 1fr;
   }
 }
 
-@media screen and (max-width: 768px) {
+@media screen and (max-width: 1002px) {
   #main {
     .container {
-      padding: 1em 20px;
+      margin: 1em 20px;
     }
 
     article {
       ul:not(.no-padding) {
-        padding: 0 20px;
+        margin: 0 20px;
       }
     }
   }
 }
+
+@media screen and (max-width: 480px) {
+  #main {
+    .container {
+      display: block;
+    }
```

Comment:
```suggestion
    .container, .has-side-nav.container {
      display: block;
    }
```

<img width="438" alt="image" src="https://github.com/nodejs/nodejs.org/assets/14062599/751545ce-8c7a-426f-9c93-6f0fabfe258d">

There is no display block on mobile devices because it prioritizes the upper `.has-side-nav.container` style